packages/react-components/react-color-picker-preview/library/src/components/ColorArea/ColorArea.types.ts

```diff
@@ -1,17 +1,32 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import * as React from 'react';
+import type { ComponentProps, ComponentState, Slot, EventHandler, EventData } from '@fluentui/react-utilities';
+
+export type ColorAreaOnColorChangeData = EventData<'click' | 'onMouseMove', React.MouseEvent> & {
+  x: number;
```

Comment:
Why do we expose the x/y values? Should the `color/value` property be there instead?

Reply:
Yes, in the following PR. I've already created a branch from this one. But this change will be applied to ColorSlider and to AlphaSlider also. So I'll make it in one PR